src/Product/WCProductAdapter.php

```diff
@@ -927,6 +928,28 @@ protected function map_gla_attributes( array $attributes ): WCProductAdapter {
 		return $this;
 	}
 
+	/**
+	 * Map the WooCommerce core global unique ID (GTIN) value if it's available.
+	 *
+	 * @since x.x.x
+	 *
+	 * @return $this
```

Comment:
I believe this should be `WCProductAdapter` ?

Reply:
Throughout Google for WooCommerce we're documenting returns as `$this` in a few places and other methods in this class do the same. So while I agree and normally would have gone with `WCProductAdapter` as it was only a small change being made I opted to go for consistency.